clang/include/clang/Basic/LangOptions.h

```diff
@@ -633,6 +633,9 @@ class LangOptions : public LangOptionsBase {
   bool AtomicFineGrainedMemory = false;
   bool AtomicIgnoreDenormalMode = false;
 
+  /// Name of the exec charset to convert the internal charset to.
+  std::string ExecCharset;
```

Comment:
Lets call that a TextEncoding consistently (replacing all instances of Codepage and Charset)

Reply:
I'm not sure if we should replace all these uses of Charset when the option name fexec-charset already has charset in it? Or do we only want the option to have that name and use encoding internally?

Reply:
The option should stay `-fexec-charset` for GCC compat, but internally we would talk about ext encoding consistently.